jwst/cube_skymatch/skycube.py

```diff
@@ -14,7 +14,7 @@
 
 
 class SkyCube():
-    """
+    r"""
```

Comment:
Just curious - for my own edification - what's the function of this "r" at the beginning of the doc string text and why is it needed now?

Reply:
Sure. It is because of the `math` mode in the docstring and escape sequences, see  https://stackoverflow.com/q/46543194